### PR TITLE
Modeltools quantgen_forecaster - avoid numeric to character conversion

### PR DESCRIPTION
**Pull request by jacobbien**
This addresses the problem described in #441 identified by @huisaddison.  This PR implements the fix that I described in my comment on that PR.  To make this PR self-contained, here's a copy of that comment:

I wanted to understand why `value` would be a factor, so I looked into this a bit, and I think I see what's happening.
 
https://github.com/cmu-delphi/covidcast/blob/6bca816b62e700cc90f31b0e3d5a6c0edda31da5/R-packages/modeltools/R/quantgen.R#L264-L271

`predict_mat` is a matrix of numeric, but when it's `cbind`-ed to a character vector, the numeric entries become character entries.  So you get "5.11329053298113" as a value.

Ok, and now here's the interesting thing.  I believe that new versions of R now changed the default of the `stringsAsFactors` argument of `data.frame` from TRUE to FALSE.  So `as.data.frame` now does not convert characters to factors by default (@huisaddison I think this explains why we were getting different results).

So the fix above would take a numeric to a character to a factor to a numeric!  I think the better fix would be to avoid the numeric to character conversion in the first place.  I think the solution is actually quite simple: Replace `cbind` with `bind_cols`.

### CODE REVIEW

**Comment by capnrefsmmat** [comment]
I'll approve once @huisaddison does, since modeltools isn't in the CODEOWNERS file and GitHub defaults to requiring review from Katie or me. Who should be listed as owners for modeltools?

File: R-packages/modeltools/R/quantgen.R
Comment (krivard) [inline R-packages/modeltools/R/quantgen.R:109]: I'm mildly surprised to find that `%>%` is not needed in this import, since it's not a built-in operator. Maybe `modeltools` is always used in an environment where `tidyr` is already loaded?

Comment (capnrefsmmat) [reply R-packages/modeltools/R/quantgen.R:109]: Looks like another function must already `@importFrom %>%`, since it's in the `NAMESPACE`: https://github.com/cmu-delphi/covidcast/blob/6bca816b62e700cc90f31b0e3d5a6c0edda31da5/R-packages/modeltools/NAMESPACE#L15

**Comment by huisaddison** [comment]
Ran locally on Ryan's vignette; works as expected.

**Review by huisaddison — APPROVED** [review]

**Comment by huisaddison** [comment]
> I'll approve once @huisaddison does, since modeltools isn't in the CODEOWNERS file and GitHub defaults to requiring review from Katie or me. Who should be listed as owners for modeltools?

@capnrefsmmat I have volunteered to be listed as maintainer for modeltools; so I guess that would be me...

**Review by capnrefsmmat — APPROVED** [review]
rubber-stamping for Addison

**Review by krivard — APPROVED** [review]
👍

**Comment by dajmcdon** [comment]
@capnrefsmmat or @krivard I think one of you has to merge this as CODEOWNERS

**Pull request merged**
